### ISSUE TITLE
The ANDI Output generated by birth month, day and year do not have the text “Person A” which all form items that have good labels on the page

### ISSUE THREAD

**Issue by ollie-iterators** [CityOfBoston/Iterators#160]
Screengrab/videos (insert file names here)
2020 09 01, Form Label Errors.mp4

Link
https://registry-certs.digital-staging.boston.gov/marriageintention?step=partnerFormA

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.6 > Chrome 85

Trusted Tester Test
Forms

Brief Description
The ANDI Output generated by birth month, day and year do not have the text “Person A” which all form items that have good labels on the page

Steps to reproduce
1)	Go to the Marriage license link and get to the second step in the process
2)	Check out the Output generated by ANDI for the form labels
3)	First Name field shows ANDI output of “Person A First Name required”
a)	Bug:  “Person A” is shown in the accessible label, but is not shown visually on the screen.  The accessibility label should match the screen instructions
b)	Note:  Additional fields have similar issues.

Desired Result
•	The form labels have all the required information to describe the output that is needed for the field

Actual Result
•	The Birth Month, Day, and Year fields do not have the “Party A” text that all the fields with good ANDI Output have.

**Comment by andrebtlima** [comment]
@phillipbentonkelly essentially what's going on here is that all the fields pertaining to Person A are labeled as such (e.g. Person A Age) except for the Month, Day, Year fields, whose labels do not indicate that they pertain to Person A. This should be corrected so that the label for the Month field (for example) reads: Person A Date of Birth, Month. So the legend is still Person A, but the ANDI output should be Date of Birth, Month. It should be  Date of Birth, Day for the Day field. It should be Date of Birth, Year for the Year field. 

This video shows what I'm talking about:

https://drive.google.com/file/d/1fUZY7wQdOSbbVnZd_aY2AaihnkVogNTZ/view

**Comment by phillipbentonkelly** [comment]
Deploying fix, it should be there soon.

**Comment by phillipbentonkelly** [comment]
@andrebtlima, this is on staging now.

**Comment by andrebtlima** [comment]
@ollie-iterators would you mind verifying that this issue has been resolved? from my own testing looks like the ANDI outputs have the correct text now, but want to make sure you guys take one more look before we mark it as complete.

**Comment by david-iterators** [comment]
The accessibility information is verified.  It was noticed that some of the terms were not consistent which might require a change in the label and therefore the accessibility information.  This is posted in https://github.com/CityOfBoston/Iterators/issues/179
